dashboard/pom.xml

```diff
@@ -26,6 +26,11 @@
 
     <artifactId>dashboard</artifactId>
     <name>Apache Uniffle Dashboard</name>
+    
+    <properties>
+        <nodeDownloadRoot>https://nodejs.org/dist/</nodeDownloadRoot>
+        <npmDownloadRoot>https://registry.npmjs.org/npm/-/</npmDownloadRoot>
```

Comment:
Is the `/-/` in the URL genuine?

Reply:
Looks like this is correct: https://github.com/eirslett/frontend-maven-plugin?tab=readme-ov-file#installing-node-and-npm

Reply:
Yes it is